scripts/src/insights.js

```diff
@@ -15,48 +15,56 @@ import open_first_section from "./modules/insights/accordion_functionality/open_
 import remove_aria_roles from "./modules/insights/accordion_functionality/remove_aria_roles";
 import remove_event from "./modules/insights/remove_event";
 import set_active from "./modules/insights/navigation/set_active";
+import set_heading_positions from "./modules/insights/accordion_functionality/set_heading_positions";
 import throttle from "./modules/throttle";
 
 document.addEventListener('DOMContentLoaded', () => {
+    add_analytics_data_card_position('.record-embed-no-image');
+    add_analytics_data_card_position('.card-group-secondary-nav');
+    audio_tracking();
+    video_tracking();
+    add_unique_ids();
+});
+
+window.addEventListener('load', () => {
     // Initialise jquery
     window.$ = $;
-    
+
     const $sectionHeadings = $(".section-separator__heading");
     const $sectionContents = $(".section-content");
     const $jumplinks = $(".jumplink");
+    let headingPositions;
 
     // These booleans are used to detect if certain enhancements (e.g. click event listeners) have been applied in order to 
     // prevent events from being added more than once.
     let mobileEnhancementsApplied = false;
     let desktopEnhancementsApplied = false;
 
-    add_analytics_data_card_position('.record-embed-no-image');
-    add_analytics_data_card_position('.card-group-secondary-nav');
-    audio_tracking();
-    video_tracking();
-    add_unique_ids();
-
     // Ids are added to sections for ARIA purposes.
     add_section_ids($sectionHeadings, $sectionContents);
-
-    if($(window).width() < 768 && !mobileEnhancementsApplied) {
+    
+    if($(window).width() < 769 && !mobileEnhancementsApplied) {
```

Comment:
A minor thing, but using `<= 768` might be more readable wherever you're doing this, especially when fixing the 768px bug in the other comment (though I appreciate it will be changed to REM anyway)